src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/_Host.cshtml

```diff
@@ -13,6 +13,7 @@
     <link href="css/site.css" rel="stylesheet" />
     <link href="BlazorServerWeb-CSharp.styles.css" rel="stylesheet" />
     <component type="typeof(HeadOutlet)" render-mode="ServerPrerendered" />
+    <link rel="shortcut icon" type="image/png" href="favicon.png"/>
```

Comment:
Could we please move this above line 11, along with the other links?

Out of curiosity, does the `favicon.png` work without this `link` attribute? We previously didn't need it because we were just using the `favicon.ico`.

Reply:
Hmm looks like we may be able to remove `shortcut` from here as it's an artifact of IE and [IE isn't officially supported by Blazor](https://docs.microsoft.com/en-us/aspnet/core/blazor/supported-platforms?view=aspnetcore-6.0).

https://www.kevinleary.net/cross-browser-favicons/